packages/babel-cli/src/babel/options.ts

```diff
@@ -213,7 +213,13 @@ export default function parseArgv(args: Array<string>): CmdOptions | null {
   const errors: string[] = [];
 
   let filenames = commander.args.reduce(function (globbed: string[], input) {
-    let files = glob.sync(input);
+    let files = process.env.BABEL_8_BREAKING
+      ? // glob 9+ no longer sorts the result, here we maintain the glob 7 behaviour
+        // https://github.com/isaacs/node-glob/blob/c3cd57ae128faa0e9190492acc743bb779ac4054/common.js#L151
+        globSync(input, { dotRelative: true }).sort(function alphasort(a, b) {
+          return a.localeCompare(b, "en");
+        })
+      : globSync(input);
```

Comment:
Instead of using the plugin, would this work?

```js
import * as glob from "glob";
```
```suggestion
    let files = process.env.BABEL_8_BREAKING
      ? // glob 9+ no longer sorts the result, here we maintain the glob 7 behaviour
        // https://github.com/isaacs/node-glob/blob/c3cd57ae128faa0e9190492acc743bb779ac4054/common.js#L151
        (USE_ESM ? glob.default.sync : glob.sync)(input, { dotRelative: true }).sort(function alphasort(a, b) {
          return a.localeCompare(b, "en");
        })
      : glob.sync(input);
```

Reply:
The USE_ESM branch should be moved to the Babel 7, otherwise it looks good to me, I will give it a try.